# msg/ThermalNodeData.msg
string name
float64 temperature
float64 heat_capacity
float64 internal_power
